server.js

```diff
@@ -1,28 +1,133 @@
 import express from 'express'
-import bodyParser from 'body-parser'
+import dotenv from "dotenv";
 import cors from 'cors'
 import mongoose from 'mongoose'
+import listEndpoints from 'express-list-endpoints'
+
+dotenv.config()
 
 const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/happyThoughts"
-mongoose.connect(mongoUrl, { useNewUrlParser: true, useUnifiedTopology: true })
+mongoose.connect(mongoUrl, { useNewUrlParser: true, useUnifiedTopology: true, useCreateIndex: true })
 mongoose.Promise = Promise
 
-// Defines the port the app will run on. Defaults to 8080, but can be 
-// overridden when starting the server. For example:
-//
-//   PORT=9000 npm start
-const port = process.env.PORT || 8080
+const port = process.env.PORT || 9000
 const app = express()
 
+//CREATES SCHEMA
+const thoughtSchema = new mongoose.Schema({
+  message: {
+    type: String,
+    required: [true, "Ding, dong! Message is required!"],
+    validate: {
+      validator: (value) => {
+        return /^[^0-9]+$/.test(value);
+      },
+      message: "Numbers are not allowed",
+    },
+    minlength: [5, 'Hey, the minimum length is 5 characters'],
+    maxlength: [140, 'Hey, the maximum length is 140 characters'],
+    unique: true,
```

Comment:
The message is not a good field to make it unique. if 3 recruiters come and send "testing" 2 of them will think it is not working 😭